portfolio/src/main/webapp/script.js

```diff
@@ -23,28 +23,58 @@ function hover(){
     }
     else{
-    	paragraphTitle = "" + info;
-    	console.log(paragraphTitle + " clicked")
+        // Clears previously active tab, changes class of newly active tab
+        var alreadyOpenTab = document.getElementsByClassName("activeTab");
+        if (alreadyOpenTab.length > 0){
+            alreadyOpenTab[0].className = alreadyOpenTab[0].className.replace(" activeTab", "");
+        }
+    	paragraphTitle = info;
+        buttonClicked.className += " activeTab";
+        // Fills content for respective tab opened
     	if (paragraphTitle == "about"){
-        	aboutMeContainer.innerText = "ABOUT Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.ABOUT "
-    	}
+        	aboutMeContainer.innerText =  aboutMe;
+            }
 		else if (paragraphTitle == "projects"){
-        	aboutMeContainer.innerText = "PROJECTS Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.PROJECTS "
-    	}
+        	aboutMeContainer.innerText = pastProjects;
+            }
 		else if (paragraphTitle == "facts"){
-       	 	aboutMeContainer.innerText = "FACTS Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.FACTS "
-    	}
-    	else if (paragraphTitle == "links"){
-        	aboutMeContainer.innerText = "LINKS Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.LINKS "
-    	}
+       	 	aboutMeContainer.innerText = funFacts;
+            }
```

Comment:
Spaces before starting braces.

Align ending braces with the `if/else if/else` keywords

Reply:
Also, align `if/else if/else`

Reply:
so this would look like:
```
if (paragraphTitle == "about") {
    aboutMeContainer.innerText =  aboutMe;
} else if (paragraphTitle == "projects") {
    aboutMeContainer.innerText = pastProjects;
} else if (paragraphTitle == "facts") {
    aboutMeContainer.innerText = funFacts;
}
```